src/routes/article-list.ts

```diff
@@ -0,0 +1,26 @@
+import Router from '@koa/router';
+import { Context, Middleware, Next } from 'koa';
+import { hydra, rdf, schema, } from 'rdf-namespaces';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.body = {
+      '@context': {
+        '@base': request.origin,
+      },
+      '@id': router.url(Routes.ArticleList),
+      '@type': hydra.Collection,
+      [hydra.title]: { '@value': 'List of articles', '@language': 'en' },
+      'http://www.w3.org/ns/hydra/core#manages': {
+        'http://www.w3.org/ns/hydra/core#property': rdf.type,
+        'http://www.w3.org/ns/hydra/core#object': schema.Article,
```

Comment:
`hydra.manages` etc not available due to https://gitlab.com/vincenttunru/rdf-namespaces/merge_requests/3#note_243892161.

Reply:
We could hold these in a separate file and use something like `hydraExtra.manages` etc to avoid the duplication and make it easier to swap out later on?

Reply:
(after reading section 5.2 of the Hydra Core Vocabulary, it all checks out)